pkgs/by-name/ka/katriawm/package.nix

```diff
@@ -16,21 +18,22 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-IWviLboZz421/Amz/QG4o8jYaG8Y/l5PvmvXfK5nzJE=";
   };
 
-  nativeBuildInputs = [
-    pkg-config
-  ];
+  nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [
     libX11
     libXft
     libXrandr
   ];
 
-  outputs = [ "out" "man" ];
+  outputs = [
+    "out"
+    "man"
+  ];
 
   strictDeps = true;
 
-  makeFlags = [ "-C" "src" ];
+  makeFlags = [ "-Csrc" ];
```

Comment:
Leave it as before. There is no need to trigger a rebuild here.

If this was generated by nixpkgs-format, then it certainly is a bug.

Reply:
Initially I've coupled them for an aesthetic reason: I believe, `-C` and `src` should belong to the same line, and `nixfmt-rfc-styles` spreads them in two lines.

Reverted.